src/baguetteBox.js

```diff
@@ -395,45 +422,50 @@
             // Remove loader element
             var spinner = document.querySelector('#baguette-img-' + index + ' .spinner');
             figure.removeChild(spinner);
-            if(!options.async && callback)
+            if (!options.async && callback) {
                 callback();
+            }
         };
         image.setAttribute('src', imageSrc);
-        if(options.titleTag && imageCaption)
+        if (options.titleTag && imageCaption) {
             image.title = imageCaption;
+        }
         figure.appendChild(image);
         // Insert caption if available
-        if(options.captions && imageCaption) {
+        if (options.captions && imageCaption) {
             figcaption.innerHTML = imageCaption;
             figure.appendChild(figcaption);
         }
         // Run callback
-        if(options.async && callback)
+        if (options.async && callback) {
             callback();
+        }
     }
 
     // Get image source location, mostly used for responsive images
     function getImageSrc(image) {
         // Set default image path from href
-        var result = imageElement.href;
+        var result = image.href;
```

Comment:
@feimosi: this one I use `image` instead of `imageElement` since that's what I thought you meant to use.